src/PrestaShopBundle/Form/Admin/Sell/Order/CartSummaryType.php

```diff
@@ -74,13 +83,21 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('payment_module', ChoiceType::class, [
                 'choices' => $this->getPaymentModuleChoices(),
-                'required' => false,
-                'placeholder' => false,
+                'required' => true,
+                'placeholder' => $this->translator->trans(
+                    '-- Choose --',
```

Comment:
So we keep a generic placeholder? Not a specific one like `Choose a payment module` and `Choose an order state` ?
@LouiseBonnard

Reply:
It's @LouiseBonnard which have decided. It's not me.

Reply:
I've browsed the software this morning and this part is messy... there are currently many ways of selecting an item in a dropdown list: 

1. the first line can be just empty, 
2. or can have the first option preselected by default (pretty risky in this case),
3. or have a specified first line ('Choose a feature', or 'Please choose a MySQL table', etc.),
4. or have a generic first line ('-- Choose --', like in the cart rule restrictions part).

My recommendation goes to the two last points for a more user-friendly experience. According to the decision we will make, we will have to specify it in the UI kit right after so that it can be taken into account in later developments - and then harmonized throughout PrestaShop.

Having a generic first line like '-- Choose --' for all dropdown selectors seems to be the easiest way. I don't mind having a specified first line, even if it multiplies the strings, but my point is that it will have to be documented in the UI kit anyway.

FYI @Junebyun, @MatShir, @prestascott, and @TristanLDD

Reply:
OK fine for me with the generic message, besides the label already explains what we are selecting Maybe we should document this decision indeed (once it's settled) for future similar behaviour

Reply:
Again, I totally agree with you, @jolelievre!
😄